src/adapters/polkadot.ts

```diff
@@ -174,7 +189,42 @@ class BasePolkadotAdapter extends BaseCrossChainAdapter {
 
     const accountId = this.api?.createType("AccountId32", address).toHex();
 
-    const [dst, acc, ass] = supportsV0V1Multilocation(this.api)
+    const doesSupportV0V1 = supportsV0V1Multilocation(this.api);
+
+    // to statemine
+    if (to === "statemine" || to === "statemint") {
+      const dst = {
+        interior: { X1: { ParaChain: toChain.paraChainId } },
+        parents: 0,
+      };
+      const acc = {
+        interior: {
+          X1: {
+            AccountId32: {
+              id: accountId,
+              network: doesSupportV0V1 ? "Any" : undefined,
+            },
+          },
+        },
+        parents: 0,
+      };
+      const ass = [
+        {
+          fun: { Fungible: amount.toChainData() },
+          id: { Concrete: { interior: "Here", parents: 0 } },
+        },
+      ];
+
+      return this.api?.tx.xcmPallet.limitedTeleportAssets(
```

Comment:
Nit: we have 2 seemingly identical implementations (in polkadot and asset hub). Would it make sense to refactor this similar to how we have a shared xtokens transfer implementation?

Reply:
I have created the issue here: #106